### ISSUE TITLE
Unable to search for any content

### ISSUE THREAD

**Issue by DualityKyle** [nixgates/plugin.video.seren#692]
When searching for content, I am getting an error. When I check the log I see:

```
2021-10-02 23:10:41.831 T:24802 WARNING <general>: SEREN (0): Kodi provided an invalid local timezone 'America/Halifax', trying a different approach
2021-10-02 23:10:41.832 T:24802 WARNING <general>: SEREN (0): Error detecting local timezone with alternative approach: module 'resources.lib.third_party.pytz' has no attribute 'tzfile'
```

I've set my timezone in Kodi, as well as in my system settings, but not change. I also get a prompt when starting Kodi that my timezone couldn't be detected.

Running Kodi 19.1.0 on Fedora 35
Seren version 2.1.4

**Comment by drinfernoo** [comment]
We need a full debug log for this one. The two lines you've included don't really give us much to go on.

**Comment by DualityKyle** [comment]
Sure thing, here you go:

[kodi.log](https://github.com/nixgates/plugin.video.seren/files/7272755/kodi.log)

**Comment by drinfernoo** [comment]
This should be fixed in 2.1.5, can you check?

**Comment by DualityKyle** [comment]
I did a fresh Kodi install and installed the new Seren version to avoid any issues with leftover cruft. Good news is I no longer get the timezone prompt. However, trying to search for any content still gives an error. I've added a new debug log below for you.

[kodi.log](https://github.com/nixgates/plugin.video.seren/files/7274361/kodi.log)

**Comment by drinfernoo** [comment]
Ok, having a look at your log looks like your build of Kodi has a more updated version of Python bundled than we're used to seeing. They've made some changes in Python 3.9 (the bundled version) that we'll need to account for.

**Comment by drinfernoo** [comment]
And it should be fixed by the next release 🎉

**Comment by DualityKyle** [comment]
I assumed it was a python version thing. That's what I get for being an early adopter on Fedora haha!

I'll patiently await the next release. Please feel free to reach out before release if you need me to test, I'll be happy to do so. Cheers!

**Comment by drinfernoo** [comment]
This particular fix should not be an issue, as the modules have been in and recommended for awhile now (since 3.3), and we just weren't using them.

Anything else (if we run into something) will probably wait until GitHub's CI engine can handle Python 3.10, as everything we push goes through automated unit testing before release... and if we can't test it, we won't release it.

**Comment by drinfernoo** [comment]
This should be fixed in 2.1.6 🎉 Can you confirm?

**Comment by DualityKyle** [comment]
It's working now!

Thanks so much for the quick turnaround! :)
